docs/code_snippet.rst

```diff
@@ -58,7 +58,10 @@ will wait for 15 minutes each time it hits the rate limit.
                yield next(cursor)
            except tweepy.RateLimitError:
                time.sleep(15 * 60)
-   
+               return
```

Comment:
I don't think the return should be here. Since this is in a `while True` loop, just sleep for 15 minutes and then it tries the call to twitter again.